src/style.css

```diff
@@ -0,0 +1,26 @@
+body,
+.first,
+.second,
+.third {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
the only one that needs resetting margins is `body`, don't apply this to everything;
paddings resetting is not necessary here at all